.github/workflows/check-binary-size.yml

```diff
@@ -6,8 +6,9 @@ name: Check binary size
 
 on:
   pull_request_target:
-    branches:
-      - master
+    # HACK(jubilee): something broke the distributed LLVM libso and I don't know what.
```

Comment:
this hack not work?

Reply:
needs to be on master for it to matter, afaik.